post/views.py

```diff
@@ -37,16 +37,11 @@ class PostListView(APIView):
 
     ### 얘네가 class inner function 들! ###
     def get(self, request):
-        posts = Post.objects.all()
-
-        serializer = PostSerializer(posts, many=True)
+        ## Post.objects.all() -> 중복되어서 나옴 why?
+        unsorted_posts = Post.objects.all()
+        sorted_posts = sorted(unsorted_posts, key= lambda p: p.like_users.count(), reverse=True)
+        serializer = PostSerializer(instance=sorted_posts, many=True)
```

Comment:
오웅 lambda 식을 사용했군요... 멋집니다..!!

아래와 같이 적어도 좋을 것 같아요~ 예시로 적어보았습니다 ㅎ.ㅎ
```suggestion
        posts = sorted(Post.objects.all(), key= lambda p: p.like_users.count(), reverse=True)
        serializer = PostSerializer(instance=posts, many=True)
```

Reply:
앗 누님 감사합니다! 확실히 이렇게 적는게 메모리를 아끼겠네요!!